app/services/prisoner_slot_availability.rb

```diff
@@ -0,0 +1,74 @@
+class PrisonerSlotAvailability
+  PRISONER_UNAVAILABLE = 'prisoner_unavailable'.freeze
```

Comment:
I'm pretty sure we have the same string defined as a constant somewhere else. We should think of a good way of sharing constants to avoid duplicates thorough the app. Just a general comment, not meant to be addressed in this PR.